minari/dataset/_storages/arrow_storage.py

```diff
@@ -7,16 +7,21 @@
 
 import gymnasium as gym
 import numpy as np
-import pyarrow as pa
-import pyarrow.dataset as ds
+
+
+try:
+    import pyarrow as pa
+    import pyarrow.dataset as ds
+except ImportError:
+    raise ImportError(
+        'pyarrow is not installed. Please install it using `pip install "minari[arrow]"`'
+    )
 
 from minari.data_collector.episode_buffer import EpisodeBuffer
 from minari.dataset.minari_storage import MinariStorage
 
 
 class ArrowStorage(MinariStorage):
-    FORMAT = "arrow"
```

Comment:
Tests fails because there are places where FORMAT is used; let's keep FORMAT.

It would be nice to assert in __init__ that the key is the same as MinariStorage.FORMAT. If you have time, change the __init__ to be similar to the of CloudStorage: use a function `get_minari_storage(key: str)` (that uses the private registry of callables) and assert that `storage.FORMAT == key`. (Then you will also need to change the places where the registry is used to use the function instead). This is a very minor problem, so it's okay to skip it.

Reply:
I should have implemented this too, however this is a bite more tricky than the CloudStorage one, since this registry is used in many tests to get the list of the available storages. What I did is creating also a function `get_storage_keys() -> List[str]` that preserve the fact of registry being private and returns the list of its keys. I replaced the previous `_storages.registry.keys()` which was in some tests and functions by this new function.

Reply:
Sounds good, can you push the changes? @JosephCarrino